openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/activity-feed-tab.less

```diff
@@ -26,26 +29,207 @@
   }
 
   .center-container {
-    flex: 0 0 calc(50% - @left-side-panel-width / 2);
-    height: @entity-details-tab-height;
+    flex: 1;
+    min-width: 0;
+    height: @user-profile-page-panel-height;
     overflow-y: scroll;
+    border-radius: 12px;
+    background-color: @white;
+    padding: 14px 20px 20px 20px;
+    margin-right: 20px;
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #eaecf5;
+    .full-width {
+      flex: 10;
+      margin-right: 0px;
+    }
   }
   .right-container {
-    flex: 0 0 calc(50% - @left-side-panel-width / 2);
-    border-left: @global-border;
-    height: @entity-details-tab-height;
+    flex: 1;
+    min-width: 0;
+    border: none;
+    height: @user-profile-page-panel-height;
     overflow-y: scroll;
+    background-color: @white;
+    border-radius: 12px;
+    border: 1px solid #eaecf5;
+    .full-width {
+      flex: 0;
+    }
 
-    .activity-feed-card-container.ant-btn:hover {
-      color: inherit;
-      background-color: @white;
+    .activity-feed-card-container {
+      margin: 0px;
+    }
+
+    .activity-feed-comments-container {
+      border-radius: 12px;
+      padding: 16px;
+      margin: 20px 0px 0px 0px;
+      border: 0.5px solid #eaecf5;
+      background: #f8f9fc;
+    }
+
+    .activity-feed-reply-button {
+      border-radius: 24px;
+      border: 1px solid #d4d4d8;
+      color: #52525b;
+      font-size: 12px;
     }
+    .comments-input-field {
+      background-color: #f4f5f7;
+      border-radius: 20px;
+      border: 1px solid #eaecf5;
+      height: 36px;
+      font-size: 12px;
+      line-height: 14.52px;
+      &::placeholder {
+        color: #a4a7ae;
```

Comment:
Can we avoid colors being used directly?

Reply:
done